libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -78,8 +78,16 @@ class AP_Airspeed
     void set_airspeed_ratio(float ratio) { set_airspeed_ratio(primary, ratio); }
 
     // return true if airspeed is enabled, and airspeed use is set
-    bool use(uint8_t i) const;
-    bool use(void) const { return use(primary); }
+    int use(uint8_t i) const;
+    void set_use(int newstate) {
```

Comment:
why doesn't this just do param[primary].use = newstate?
even better to use set_and_notify, so the GCS gets the update

Reply:
I will update it to use set_and_notify as I've noticed that the its annoying when the GCS is not updated with this change in state.

Reply:
Changed it to use the set_and_notify, thanks for that suggestion.  Far easier to test when the gcs has the right parameter values.